pybossa/api/task_run.py

```diff
@@ -84,6 +84,13 @@ def check_can_post(self, project_id, task_id):
         if not can_post(project_id, task_id, get_user_id_or_ip()):
             raise Forbidden("You must request a task first!")
 
+    def _custom_filter(self, filters):
```

Comment:
Is this function called somewhere? I didn't find any change that calls this function. Also I suppose we don't need `return filters` since the parameter should be called using pointer.

Reply:
Its part of base class and can be override by derived class if needed. its called [here](https://github.com/bloomberg/pybossa/pull/307/files#diff-36e1d23f11e76e2143b66764197f5db4R267)